public/components/player/edit/edit_test.js

```diff
@@ -57,10 +55,30 @@ QUnit.module('player/edit', function(hooks){
 					vm.unbind("saved");
 					done();
 				})
-				vm.savePlayer()
+				vm.savePlayer();
 			
 		});
 
+		QUnit.test("Create player fails without name", function(assert){
+			assert.expect(2);
+			var done = assert.async(),
+				player = {
+					"weight": 200,
+					"height": 71,
+					"birthday": "1980-01-01"
+				},
+				playerModel = new Player(player),
+				vam = new ViewModel({
+					player: playerModel
+				});
+
+				vm.savePlayer().fail(function(resp, type){
```

Comment:
we shouldn't rely on `savePlayer` returning a promise b/c the template wouldn't.  Instead we should probably call:

```
vm.savePlayer();
vm.attr("savePromise").then(function(){...}, function(){...})
```